packages/mdc-menu/foundation.ts

```diff
@@ -99,9 +100,7 @@ export class MDCMenuFoundation extends MDCFoundation<MDCMenuAdapter> {
     // Wait for the menu to close before adding/removing classes that affect styles.
     this.closeAnimationEndTimerId_ = setTimeout(() => {
       const selectionGroup = this.getSelectionGroup_(listItem);
-      if (selectionGroup) {
-        this.handleSelectionGroup_(selectionGroup, index);
-      }
+      this.handleSelectionGroup_(selectionGroup, index);
```

Comment:
I feel this shouldn't even call `handleSelectionGroup_` if the current item is not in selection group.